<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditWing</class>
 <widget class="QDialog" name="DlgEditWing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit wing properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="textLabel6">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>leName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cbAutoSym">
          <property name="text">
           <string>Automatic symmetry</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rbLinear">
          <property name="text">
           <string>Linear interpolation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbDetectWinglet">
          <property name="text">
           <string>Autodetect winglet</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="rbCubic">
          <property name="text">
           <string>Bicubic surface</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbVisible">
          <property name="text">
           <string>Visible</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pbChangeColor">
          <property name="text">
           <string>Change color...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Right cap height</string>
            </property>
            <property name="buddy">
             <cstring>sbRightCap</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbRightCap">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Left cap height</string>
            </property>
            <property name="buddy">
             <cstring>sbLeftCap</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbLeftCap">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sections</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QListWidget" name="listSection">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="pbNewSection">
          <property name="text">
           <string>&amp;New section</string>
          </property>
          <property name="shortcut">
           <string>Alt+N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRemoveSection">
          <property name="text">
           <string>&amp;Remove section</string>
          </property>
          <property name="shortcut">
           <string>Alt+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbEditSection">
          <property name="text">
           <string>&amp;Edit section</string>
          </property>
          <property name="shortcut">
           <string>Alt+E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbMoveUp">
          <property name="text">
           <string>Move up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbMoveDown">
          <property name="text">
           <string>Move down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSortSections">
          <property name="text">
           <string>Sort sections</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Transformation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel7">
          <property name="text">
           <string>Position x</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScienceSpinBox" name="sbPosX">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel7_2">
          <property name="text">
           <string>Position y</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QScienceSpinBox" name="sbPosY">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel7_3">
          <property name="text">
           <string>Position z</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbPosZ</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QScienceSpinBox" name="sbPosZ">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel11">
          <property name="text">
           <string>Rotation X</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbRotX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScienceSpinBox" name="sbRotX">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel10">
          <property name="text">
           <string>Rotation Y</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbRotY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QScienceSpinBox" name="sbRotY">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel11_2">
          <property name="text">
           <string>Rotation Z</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>sbRotZ</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QScienceSpinBox" name="sbRotZ">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pbStat">
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>111</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbTransform">
       <property name="text">
        <string>&amp;Transform</string>
       </property>
       <property name="shortcut">
        <string>Alt+Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>cbAutoSym</tabstop>
  <tabstop>cbDetectWinglet</tabstop>
  <tabstop>cbVisible</tabstop>
  <tabstop>rbLinear</tabstop>
  <tabstop>rbCubic</tabstop>
  <tabstop>pbChangeColor</tabstop>
  <tabstop>sbRightCap</tabstop>
  <tabstop>sbLeftCap</tabstop>
  <tabstop>listSection</tabstop>
  <tabstop>pbNewSection</tabstop>
  <tabstop>pbRemoveSection</tabstop>
  <tabstop>pbEditSection</tabstop>
  <tabstop>pbMoveUp</tabstop>
  <tabstop>pbMoveDown</tabstop>
  <tabstop>pbSortSections</tabstop>
  <tabstop>sbPosX</tabstop>
  <tabstop>sbPosY</tabstop>
  <tabstop>sbPosZ</tabstop>
  <tabstop>sbRotX</tabstop>
  <tabstop>sbRotY</tabstop>
  <tabstop>sbRotZ</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>pbStat</tabstop>
  <tabstop>pbTransform</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditWing</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutoSym</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbDetectWinglet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
